course05/homework01/skeleton_ivo/string.h

```diff
@@ -0,0 +1,30 @@
+#include <string>
+#include <ostream>
+#include <istream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
+	String(std::istream&);
+
+	bool Empty() const;
+	std::size_t Size() const;
+
+	bool operator==(const String) const;
+	bool operator!=(const String) const;
+
+	String Substring(int start) const;
+	String Substring(int start, int length) const;
+
+	std::string str() const
+	{
+		return mString;
+	}
+
+private:
+	std::string mString;
```

Comment:
The string was supposed to be on the stack; this isn't the case of `std::string`!